.travis.yml

```diff
@@ -16,9 +16,14 @@ matrix:
       script: ci/run.sh
     - install: true
       script: ci/dox.sh
-
+    - env: CLIPPY=On TARGET=x86_64-unknown-linux-gnu NO_ADD=1
```

Comment:
I think the `CLIPPY` env var isn't needed here?

Reply:
Er nvmd, this is needed to distinguish it in `allow_failures`

Reply:
yes that's right, I should add a comment, I did the same for the RUSTFMT=On variable, I just left it there so that one can tell the builds appart, maybe there is a better way to name the builds?